services/web-app/components/mini-card/mini-card.module.scss

```
@@ -0,0 +1,50 @@
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/type' as type;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+@use '@carbon/react/scss/motion' as motion;
+
+@media screen and (prefers-reduced-motion: reduce) {
+  .card {
+    display: flex;
+    height: convert.rem(64px);
+    flex-direction: column;
+    justify-content: center;
+    background: theme.$layer-01;
+    transition: none;
+  }
+}
```

```suggestion
@media screen and (prefers-reduced-motion: reduce) {
  .card {
    transition: none;
  }
}
```

I think we can simplify this